identity/src/main/java/com/android/identity/GattServer.java

```diff
@@ -148,18 +165,59 @@ boolean start() {
             mCharacteristicIdent = c;
         }
 
+        // Offers support to L2CAP when we have UUID characteristic and the OS version support it
+        mSupportL2CAP = (mCharacteristicL2CAPUuid != null) && (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q);
```

Comment:
Suggest to call it `mUsingL2CAP` instead (and also update logging statement below).

Reply:
Done